pq-crypto/s2n_pq_asm.h

```diff
@@ -15,4 +15,15 @@
 
 #pragma once
 
+#ifndef S2N_BLOCK_NONPORTABLE_OPTIMIZATIONS
+#define S2N_BLOCK_NONPORTABLE_OPTIMIZATIONS 0
+#endif
+
 #define S2N_ANY_NONPORTABLE_OPTIMIZATIONS_ENABLED   __AVX__ || __AVX2__ || __BMI2__
+
+#if S2N_BLOCK_NONPORTABLE_OPTIMIZATIONS && S2N_ANY_NONPORTABLE_OPTIMIZATIONS_ENABLED
```

Comment:
Should `S2N_ANY_NONPORTABLE_OPTIMIZATIONS_ENABLED` be in parenthesis, else your macro expands to:

`S2N_BLOCK_NONPORTABLE_OPTIMIZATIONS && __AVX__ || __AVX2__ || __BMI2__`